src/Homework2/task2.py

```diff
@@ -0,0 +1,19 @@
+'''
+2. Найти самое длинное слово в введенном предложении.
+Учтите что в предложении есть знаки препинания.
+Подсказки:
+-	my_string.split([chars]) возвращает список строк.
+-	len(list) - количество элементов в списке
+Задачу поместите в файл task2.py в папке src/homework2.
+
+'''
+import re
+while True:
+    sentence = str(input("Введите строку: "))
+    sentence_cleared = re.sub('[^ A-Za-z0-9]', '', sentence)
```

Comment:
what about $ symbol and many others?

Reply:
All characters except A-Za-z0-9 are removed 
'^' - except